memcached/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM debian:jessie
```

Comment:
sounds like a good plan, I'll switch to building an image and working from that. That Dockerfile is mostly from the official memcached image, I just edited to add the containerbuddy stuff. I am going to go back to it and try and build it from the original image rather than copying the dockerfile the way I did here. I originally had a problem with the way they are using the memcached user, but I needed to do more work as root. Now I think I can switch back to root, install my needed things and then to back to memcached user